modules/unsupported/mongodb/README.rst

```diff
@@ -2,17 +2,128 @@ MongoDB DataStore
 =================
 
 Installing MongoDB
--------------------------
+------------------
 
-Install MongoDB using the appropriate package from:
-http://www.mongodb.org/downloads
+Install MongoDB using appropriate package from: http://www.mongodb.org/ downloads.
 
-The MongoDB plugin depends on a JavaScript MapReduce script to calculate schema. The script (MetaDataCompute.js), and installation specifics (INSTALL.txt), can be found in modules/unsupported/mongodb/src/main/javascript/. Install this script on your MongoDB server.
+To publish a collection:
+
+* A compliant collection must have one or more spatial indexes of type 2dsphere. Any collection without an indexed field of type 2dsphere will be ignored.
+
+* When examining compliant collections, the schema store will be queried for an existing schema by using the collection name as the schema name. If a schema exists it is utilized otherwise it is inferred from the collection and cached to the schema store for reuse.
+
+* A collection must have its underlying coordinate system set to EPSG:4326 (WGS84, latitude/longitude order).
+
```

Comment:
The wording here is a little confusing...  At first read I interpreted this to be a MongoDB collection property when I believe it refers to the GeoTools DataStore instance property for CRS (when I worked on this WGS84 for MongoDB 2dsphere indices was required and implicit so there wasn't a MongoDB collection level property).  Also confusing is mention of "latitude/longitude" as the coordinate when the ordering required by GeoJSON geometries is "longitude/latitude" (ignore that GeoJSON can use OGC urn for CRS and *not* respect authority defined axis ordering...)